gulpfile.js

```diff
@@ -60,6 +60,17 @@ function test()
     return runTests();
 }
 
+function insertPreventDuplicateLoadingCheck(fileName) {
```

Comment:
@dapperdrop @allmywant - minor issue, but I wonder if there's a clearer way to label this function. Like `insertDuplicateContainerCheck`.